packages/icon/src/createGlyphComponent.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+export function getGlyphTitle(name: string, title?: string | null) {
+  if (title === null) {
```

Comment:
This may be too late but this seems like weird implicit behavior. null and undefined are almost always used interchangeably. What if we had this behave if title was `false` instead?